tsconfig.esm.base.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "module": "ES2022",
+    "module": "Node16",
```

Comment:
beware with this change. Here https://www.typescriptlang.org/tsconfig/#module is stated that this changes the moduleResolution. I think we have to be sure that the compiled code is the one we expect

Reply:
Yes you are right I had to change this to make the build but now it is failing some tests in core related to regexp. I have to check better how the build is made